hw4/tasks/task4.py

```diff
@@ -16,3 +16,20 @@
 
 > Подсказка: Используйте оператор +=
 """
+
+alphabet = 'абвгдеёжзийклмнопрстуфхцчшщъыьэюя'
+
+phrase = alphabet[-1].upper() # Я
+phrase += " "
+phrase += alphabet[9:7:-1] # из
```

Comment:
только комменты по PEP 8 должны идти после 2 пробелов
```suggestion
phrase += alphabet[9:7:-1]  # из
```